beets/importer.py

```diff
@@ -1352,7 +1358,26 @@ def emitter(task):
         ])
         return pipeline.multiple(ipl.pull())
 
-    resolve_duplicates(session, task)
+    if type(task) != MergedImportTask:
```

Comment:
I know this seems crazy, but just to check: is this condition actually necessary? That is, if we merge a bunch of albums into one, would it be OK to check again for duplicates of the newly-tagged merged album?

Reply:
I did this because the initially detected duplicates weren't removed, hence would again be marked in this MergedImportTask's duplicate list and give a second prompt to the user. Choosing "Remove" at this stage would be the correct way to go, which is basically what this condition does automatically. Also it helps avoid another pointless check for duplicates which we know exist.

Is there a way to remove those albums from the library, so that they don't get detected as duplicates again? like probably setting the album's id to `None` ?

Reply:
Aha, got it. This makes me think that there might be a slightly deeper issue to address here: we probably shouldn't detect a duplicate when the "duplicate" album actually consists of the same songs as the newly-imported album! This should probably be true for any re-import situation, not just the new functionality you've added here.

In particular consider this check from `find_duplicates`:

https://github.com/beetbox/beets/blob/9c6910d1675926d9055508f4f67683cda86f9179/beets/importer.py#L637-L639

This says that we don't detect a duplicate if the set of songs is *exactly* the same between the old and new album. Perhaps this should be changed to a *subset* check instead. Does that seem reasonable?

Reply:
A subset check would be correct. I will make that change because the subset case would only occur in merge scenario. We also wouldn't need a separate MergeImportTask class and check too.